src/main/sensors/current.h

```diff
@@ -65,8 +65,8 @@ typedef struct currentMeterADCState_s {
 } currentMeterADCState_t;
 
 typedef struct currentSensorADCConfig_s {
-    int16_t scale;              // scale the current sensor output voltage to milliamps. Value in 1/10th mV/A
-    int16_t offset;            // offset of the current sensor in millivolt steps
+    float scale;              // scale the current sensor output voltage to milliamps. Value in mV/A
```

Comment:
These are `int16_t` because CLI and MSP are only set up to handle up to `int16_t`, so they should stay that way.